src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -102,6 +111,7 @@ def run_webhooks_and_report_status(
                 webhook.key,
             )
             privacy_request.error_processing(db)
+            privacy_request.cache_failed_checkpoint_details(current_step)
```

Comment:
Using our new request execution checkpoints this lets us resume from the pre or post execution webhooks step later if they fail.  We used to be able to resume from pause but had no handling to resume from failure.